### PR TITLE
Dist windows service

### PR DESCRIPTION
**Pull request by bobchaos**
## Description
More work towards https://github.com/chef/chef/issues/8376 this one covers the windows_service and windows_service_manager logs/descriptions. I'm also sneaking in some missing trademarks I stumbled on.

## Related Issue
https://github.com/chef/chef/issues/8376

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [X] I have read the **CONTRIBUTING** document.
- [X] I have run the pre-merge tests locally and they pass.
- [NA] I have updated the documentation accordingly.
- [NA] I have added tests to cover my changes.
- [X] All new and existing tests passed.
- [X] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: chef-bin/bin/chef-service-manager [hunk 1]
```diff
@@ -24,9 +24,9 @@ require "chef/application/windows_service_manager"
 
 if Chef::Platform.windows?
   chef_client_service = {
-    service_name: "chef-client",
-    service_display_name: "Chef Client Service",
-    service_description: "Runs Chef Client on regular, configurable intervals.",
+    service_name: Chef::Dist::CLIENT,
```
Comment (tas50) [inline chef-bin/bin/chef-service-manager]: This file needs to require dist

File: lib/chef/application.rb [hunk 2]
```diff
@@ -343,7 +343,7 @@ def fork_chef_client
           exit 0
         end
       end
-      logger.trace "Fork successful. Waiting for new chef pid: #{pid}"
+      logger.trace "Fork successful. Waiting for new #{Chef::Dist::EXEC} pid: #{pid}"
```
Comment (tas50) [inline lib/chef/application.rb]: this should actually be chef-client

File: lib/chef/cookbook_site_streaming_uploader.rb [hunk 3]
```diff
@@ -23,6 +23,8 @@
 require "mixlib/authentication/signedheaderauth"
 require "openssl" unless defined?(OpenSSL)
```
Comment (tas50) [inline lib/chef/cookbook_site_streaming_uploader.rb]: no need for this extra line

File: lib/chef/cookbook_site_streaming_uploader.rb [hunk 4]
```diff
@@ -36,7 +38,7 @@ class CookbookSiteStreamingUploader
     class << self
 
       def create_build_dir(cookbook)
-        tmp_cookbook_path = Tempfile.new("chef-#{cookbook.name}-build")
+        tmp_cookbook_path = Tempfile.new("#{Chef::Dist::EXEC}-#{cookbook.name}-build")
```
Comment (tas50) [inline lib/chef/cookbook_site_streaming_uploader.rb]: I think what we need is some sort of "shortname" value. If we could do these over again they probably would be chef-client-whatever, but it's too late to change that.

**Comment by bobchaos** [comment]
closing in favor of https://github.com/chef/chef/pull/9060

**Pull request closed without merging**